x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/forcemerge.tsx

```diff
@@ -73,23 +82,44 @@ export const Forcemerge: React.FunctionComponent<Props> = ({
       <EuiSpacer />
       <div id="forcemergeContent" aria-live="polite" role="region">
         {phaseData.forceMergeEnabled ? (
-          <ErrableFormRow
-            id={`${phase}-selectedForceMergeSegments`}
-            label={i18n.translate('xpack.indexLifecycleMgmt.warmPhase.numberOfSegmentsLabel', {
-              defaultMessage: 'Number of segments',
-            })}
-            isShowingErrors={isShowingErrors}
-            errors={errors?.selectedForceMergeSegments}
-          >
-            <EuiFieldNumber
-              data-test-subj={`${phase}-selectedForceMergeSegments`}
-              value={phaseData.selectedForceMergeSegments}
-              onChange={(e) => {
-                setPhaseData('selectedForceMergeSegments', e.target.value);
-              }}
-              min={1}
-            />
-          </ErrableFormRow>
+          <>
+            <ErrableFormRow
+              id={`${phase}-selectedForceMergeSegments`}
+              label={i18n.translate('xpack.indexLifecycleMgmt.forceMerge.numberOfSegmentsLabel', {
+                defaultMessage: 'Number of segments',
+              })}
+              isShowingErrors={isShowingErrors}
+              errors={errors?.selectedForceMergeSegments}
+            >
+              <EuiFieldNumber
+                data-test-subj={`${phase}-selectedForceMergeSegments`}
+                value={phaseData.selectedForceMergeSegments}
+                onChange={(e) => {
+                  setPhaseData('selectedForceMergeSegments', e.target.value);
+                }}
+                min={1}
+              />
+            </ErrableFormRow>
+            <EuiFormRow
+              hasEmptyLabelSpace
+              helpText={
+                <FormattedMessage
+                  id="xpack.indexLifecycleMgmt.editPolicy.forceMerge.bestCompressionText"
+                  defaultMessage="Uses DEFLATE for a higher compression ratio but slower stored fields performance
+                  than the default LZ4 codec."
```

Comment:
Some alternate suggestions in order of preference:

* `Better compress stored fields at the cost of slower performance.`
* `Use higher compression for stored fields at the cost of slower performance.`
* `Use the DEFLATE codec for higher compression but slower stored fields performance. If disabled, the default LZ4 codec is used.`